Oppgave 2.ipynb

```diff
@@ -0,0 +1,1048 @@
+{
```

Comment:
<p>Er det ikke motsatt her? Høyere presisjon for kvinner enn menn?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Bsingstad/DMBA2024/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>